### ISSUE TITLE
Need a function to generate caves

### ISSUE THREAD

**Issue by slemonide** [slemonide/gen#6]

**Comment by HybridDog** [comment]
something like this?
https://forum.minetest.net/viewtopic.php?p=180886#p180886

**Comment by slemonide** [comment]
Yes. Are they connected to the ground?

**Comment by slemonide** [comment]
I was thinking to make all caves connected, so they would look like mazes.

**Comment by HybridDog** [comment]
l think they're endless

**Comment by slemonide** [comment]
Can you commit them?

**Comment by HybridDog** [comment]
yes, l did a pull request: https://github.com/slemonide/gen/pull/7

Edit: maybe your gen mod could be used for the fractured game…

**Comment by slemonide** [comment]
Sure.

**Comment by slemonide** [comment]
Those caves are magnificent!

**Comment by slemonide** [comment]
Probably an ore generator would also come in handy. What do you think, should we just merge the default one or do something creative?

**Comment by HybridDog** [comment]
> Those caves are magnificent!

thanks

> Probably an ore generator would also come in handy. What do you think, should we just merge the default one or do something creative?

l think about a combination of both: reducing ore rarity with ws functions
